### PR TITLE
First Pull request for portfolio project

### PR DESCRIPTION
**Pull request by prius2055**
This is a pull request for the first part of the portfolio project from Microverse

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <h1>John Doe</h1>
```
[the remark this answers is not in the document]
Comment (prius2055) [reply index.html]: Thanks, I have worked on this

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <div class="hero-section-icons">
+            <img src="/icons/github.png" alt="github icon" />
+            <img src="/icons/linkedin.png" alt="linkedin icon" />
+            <img src="/icons/angel-list.png" alt="angel list icon" />
+            <img src="/icons/twitter.png" alt="twitter icon" />
+            <img src="/icons/medium.png" alt="medium icon" />
+          </div>
```
[the remark this answers is not in the document]
Comment (prius2055) [reply index.html]: Also resolved. Thank you

File: index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        <section class="hero-section">
+          <h1>Hey There. I’m Alex</h1>
+          <h2>I'm a software developer</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <div class="hero-section-icons">
+            <img src="/icons/github.png" alt="github icon" />
```
[the remark this answers is not in the document]
Comment (prius2055) [reply index.html]: Ok. This has been fixed, thank you

File: README.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
[the remark this answers is not in the document]
Comment (prius2055) [reply README.md]: The MIT license has been updated

File: README.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+I would like to thank the Microverse team for their support
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (iambenkis) [inline README.md:150]: - [ ] As requested by the previous you should remove this note, it will make your README file more professional :+1:

**Review by iambenkis — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @prius2055  :wave:

### Highlights :1st_place_medal:

All linter checks passed :heavy_check_mark:
Descriptive pull request :heavy_check_mark:
Used GitHub flow :heavy_check_mark:
UI design  :heavy_check_mark: 

Good job on implementing most of the requirements but there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @iambenkis in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 5 above)
Comment (prius2055) [reply README.md]: This line has been removed

**Review by muneebulrehman — APPROVED** [review]
Hi @prius2055 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewers.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**